### ISSUE TITLE
Greetings! May I ask, how to use odeavg.f90?

### ISSUE THREAD

**Issue by CRquantum** [WarrenWeckesser/dde_solver#2]
Dear @jacobwilliams ,

The odepack stuff is great! 
I see that you have made an independent odeavg.f90 file. 
I see there is some instructions in it, but I am still not very clear as how do I use this file.
For example, I added 
```
use DDE_SOLVER_M 
```
in the function so it can recognize some stuff from module DDE_SOLVER_M. 
like below, 
![image](https://user-images.githubusercontent.com/52009942/149642525-e41ef8b1-874e-4259-9e9f-32a46018a217.png)

However, for example, I compile,  odeavg.f90+ DDE_SOLVER_M.f90+c2g.f90
it gives me some error like below, 
![image](https://user-images.githubusercontent.com/52009942/149642559-577df3db-5840-4012-b620-31cdad9d42c4.png)

In short, when you have time, would it be possible to provide an example on how to use odeavg.f90?
Thank you very much indeed!

Best regards,
Rong

**Comment by WarrenWeckesser** [comment]
@CRquantum, sorry for not responding sooner (but note that I am not @jacobwilliams!).

I have never used the `odeavg.f90` code.  In fact, here's a comment from an email from Skip Thompson, the author of the `dde_solver` Fortran library:

> I really think that odeavg should just be deleted from the solver permanently. No one ever used it (except me for the examples in a paper Larry and I wrote).

I kept the code in a separate file for posterity, but there are no plans to maintain or support it.  In fact, I just pushed the file down into an 'archive' directory (and I should probably just delete it).

**Comment by CRquantum** [comment]
Great thank you!
